clang/lib/Basic/Attributes.cpp

```diff
@@ -166,7 +166,8 @@ getScopeFromNormalizedScopeName(StringRef ScopeName) {
       .Case("hlsl", AttributeCommonInfo::Scope::HLSL)
       .Case("msvc", AttributeCommonInfo::Scope::MSVC)
       .Case("omp", AttributeCommonInfo::Scope::OMP)
-      .Case("riscv", AttributeCommonInfo::Scope::RISCV);
+      .Case("riscv", AttributeCommonInfo::Scope::RISCV)
+      .Default(AttributeCommonInfo::Scope::NONE);
```

Comment:
Won't it be surprising that `""` and `"wobble"` will both come back as `NONE`? I would have expected the latter to return something like "Unknown" so it's distinguished from no scope name.

Reply:
I added `UNKNOWN` kind as the default case